bigbluebutton-html5/imports/ui/components/recording/component.jsx

```diff
@@ -36,11 +40,13 @@ const propTypes = {
   intl: intlShape.isRequired,
   closeModal: PropTypes.func.isRequired,
   toggleRecording: PropTypes.func.isRequired,
+  recordingTime: PropTypes.number,
   recordingStatus: PropTypes.bool,
   amIModerator: PropTypes.bool,
 };
 
 const defaultProps = {
+  recordingTime: undefined,
```

Comment:
If recordingTime is supposed to be a number, shouldn't the default also be a number?